ui/pages/confirmations/hooks/alerts/useBlockaidAlerts.ts

```diff
@@ -136,13 +135,7 @@ const useBlockaidAlerts = (): Alert[] => {
     }
 
     return [normalizeProviderAlert(securityAlertResponse, t, reportUrl)];
-  }, [
-    isTransactionTypeSupported,
-    isResultTypeIgnored,
-    securityAlertResponse,
-    stringifiedJSONData,
-    t,
-  ]);
+  }, [signatureSecurityAlertResponse, transactionSecurityAlertResponse, t]);
```

Comment:
`stringifiedJSONData` was not valid dependency as it is not saved across rendering cycles. I minimised dependencies to variable that are preserved across rendering cycles.